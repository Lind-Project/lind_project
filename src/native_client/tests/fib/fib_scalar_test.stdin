rpc fib i(1) i(1) i(5) * i(0)
rpc fib i(2) i(3) i(10) * i(0)
rpc fib i(1) i(1) i(15) * i(0)
