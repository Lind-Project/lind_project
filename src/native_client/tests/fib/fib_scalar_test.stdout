rpc call initiated fib:iii:i
input 0:  i(1)
input 1:  i(1)
input 2:  i(5)
rpc call complete fib:iii:i
output 0:  i(8)

rpc call initiated fib:iii:i
input 0:  i(2)
input 1:  i(3)
input 2:  i(10)
rpc call complete fib:iii:i
output 0:  i(233)

rpc call initiated fib:iii:i
input 0:  i(1)
input 1:  i(1)
input 2:  i(15)
rpc call complete fib:iii:i
output 0:  i(987)

